scripts/gbp_format_changelog

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# Simple changelog entry formatter
+#
+# It simply uses the built in formatter and linewraps the text
+#
+# Use git-dch --customizations=/usr/share/doc/git-buildpackage/examples/wrap_cl.py
+# or set it via gbp.conf
+
+import textwrap
+import re
+import gbp.dch
+
+
+FILTER_CLOUD_INIT_UPSTREAM_COMMITERS = (
+    "Chad Smith", "James Falcon", "Brett Holman"
+)
+
+FILTER_NOISY_COMMIT_REGEX = (
+  r'update changelog.*',
+  r'refresh patches against.*',
+)
+
+UNWRAPPABLE_DELIMITERS = {
+  "]": "[",
+  ")": "(",
+}
+
+
+def _wrap_on_delimiter(text, prefix="", max_length=70) -> list:
+    """Break lines at specific UNWRAPPABLE_DELIMITERS.
+
+    When a line ends with either (LP: #XXX) or [Author Name] avoid using
+    textwrap.wrap which breaks at the last whitespace.
+
+    Instead break at the leading ( or [ special delimiter to ensure entire
+    author name or LP bug reference remains on the same line.
+
+    Fallback to use textwrap.wrap if special conditions don't apply.
+
+    Return a list of individual lines.
+    """
+    if len(text) <= max_length:
+        return [prefix + text]
+    if text[-1] in UNWRAPPABLE_DELIMITERS:
+        delimiter = UNWRAPPABLE_DELIMITERS[text[-1]]
+        part1, sep, part2 = text.rpartition(delimiter)
+        remainder = ""
```

Comment:
```suggestion
```
Variable not used, please remove.

Reply:
I added a #92 to better track/fix these type of errors per commit for me so I don't introduce flake, isort or black formatting errors in any new scripts.